From 05f427b99359758386c0dae792966952ecff00ba Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sat, 3 Jan 2026 15:14:56 +0800
Subject: [PATCH 37/49] loonggpu: conftest: add -fms-extensions to CFLAGS

Since Linux 6.19, some constructs in the (non-UAPI) headers uses the
Go-like unnamed member, that -fms-extensions enables.

Link: https://git.kernel.org/torvalds/c/c4781dc3d1cf
Signed-off-by: Xi Ruoyao <xry111@xry111.site>
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 conftest.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/conftest.sh b/conftest.sh
index 1001ea4..436c73a 100755
--- a/conftest.sh
+++ b/conftest.sh
@@ -89,7 +89,8 @@ build_cflags() {
     BASE_CFLAGS="-O2 -D__KERNEL__ \
 -DKBUILD_BASENAME=\"#conftest$$\" -DKBUILD_MODNAME=\"#conftest$$\" \
 -nostdinc -isystem $ISYSTEM \
--Wno-implicit-function-declaration -Wno-strict-prototypes -std=gnu11"
+-Wno-implicit-function-declaration -Wno-strict-prototypes -std=gnu11 \
+-fms-extensions"
 
     if [ "$OUTPUT" != "$SOURCES" ]; then
         OUTPUT_CFLAGS="-I$OUTPUT/include2 -I$OUTPUT/include"
-- 
2.56.0

